cpp/tests/reductions/scan_tests.cpp

```diff
@@ -428,6 +428,41 @@ TEST_F(ScanStringsTest, MoreStringsMinMax)
   this->scan_test(v, b, make_max_aggregation(), scan_type::EXCLUSIVE, null_policy::EXCLUDE);
 }
 
+template <typename T>
+struct ScanChronoTest : public BaseFixture {
+};
+
+TYPED_TEST_CASE(ScanChronoTest, ChronoTypes);
+
+TYPED_TEST(ScanChronoTest, ChronoMinMax)
+{
+  cudf::test::fixed_width_column_wrapper<TypeParam, int32_t> col({5, 4, 6, 0, 1, 6, 5, 3},
+                                                                 {1, 1, 1, 0, 1, 1, 1, 1});
+  cudf::test::fixed_width_column_wrapper<TypeParam, int32_t> expected_min({5, 4, 4, 0, 1, 1, 1, 1},
+                                                                          {1, 1, 1, 0, 1, 1, 1, 1});
```

Comment:
Just out of curiousity, is the expected 4th element '0' just because the input is null?   Would the output be the same if the input was:
```
  cudf::test::fixed_width_column_wrapper<TypeParam, int32_t> col({5, 4, 6, 99, 1, 6, 5, 3},
                                                                 {1, 1, 1, 0, 1, 1, 1, 1});
```

Reply:
Yes.